ee_plugin.py

```diff
@@ -13,6 +13,7 @@
 import webbrowser
 from builtins import object
 import os.path
+import json
```

Comment:
See the style guide on [import](https://www.python.org/dev/peps/pep-0008/#imports) order

```
import json
import os.path
import webbrowser

from builtins import object
```
Also note that [`pathlib`](https://docs.python.org/3/library/pathlib.html) provides a more richer api to paths than the older `os.path`